### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Aichnerc**
Countless changes have been made to further improve the readme.

**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the README.md file is not up to our standards.

**What is the new behavior (if this is a feature change)?**
- The README.md is now up to our standards.

### CODE REVIEW

**Review by schettn — APPROVED** [review]
Yep

**Review by Theralun — APPROVED** [review]
Looks good.

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

File: README.md [hunk 1]
```diff
@@ -93,23 +102,24 @@ those rules whenever possible.
...
+    		<td>
+			<a href="https://github.com/aichnerc">
+				<img src="https://avatars.githubusercontent.com/aichnerc?s=100" alt="Avatar aichnerc">
+			</a>
+		</td>
+	</tr>
+  <tr>
+    		<td><a href="https://github.com/aichnerc">Christian Aichner</a></td>
+	</tr>
+</table>
```
Comment (kleberbaum) [inline README.md:116]: Are this indentations intended?

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

File: README.md (hunk 1 above)
Comment (Aichnerc) [reply README.md:116]: image

**Pull request merged**